.github/pull_request_template.md

```diff
@@ -0,0 +1,46 @@
+<!--- PR title format: [GH#<gh-issue-id>][BZ#<bz-issue-id>][OSDOCS-<jira-issue-id>]: <short-description-of-the-pr> --->
+
+Version(s):
+<!--- Specify the version or versions of OpenShift your PR applies to.  --->
+
+Issue:
+<!--- Add a link to the Bugzilla, Jira, or GitHub issue. --->
+
+Link to docs preview:
+<!--- Add direct links to the preview build.
+The preview will be generated after you open the PR.
+You will need to edit the comment to add the link after the preview builds.--->
+
+Additional information:
+<!--- Optional: Include additional context or expand the description here.
+
+Next steps after opening your PR:
+
+* Ask for peer review from the OpenShift docs team:
```

Comment:
@bburt-rh Do you have a link to the DevTools process? I am concerned that adding alternative steps will make things more confusing and this is quite long already. WDYT of adding a link to your contributor docs for DevTools?

Reply:
@michaelryanpeter You're right that it might make things too complex to include variations. Most dev teams probably don't even know, or care, whether the core OCP docs team or an aligned team like DevTools docs is doing the doc work. Forcing them to care about how we divvy up the work is probably not optimal. 

I'm mostly trying to keep things from slipping through the cracks -- for instance, if a monitoring dev submitted a PR to the main OCP docs review channel, I'd like to be able to review it as well because I know the monitoring docs well and I want to be aware of any changes. 

Maybe it's an internal education thing for the core OCP docs review team to know to coordinate docs PRs for aligned projects to the appropriate aligned team? Or am I overthinking this whole process? Simple instructions are best...

Reply:
@bburt-rh You raise a good point. Maybe we can tackle this as a separate issue? I know there is some work happening around peer review with core and aligned teams. Maybe that is the place for a discussion on how to deal with dev submitted PRs for aligned products?

Then we can revise this PR template or update the way the core peer review squad handles these types of situations? WDYT?